src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -4,8 +4,9 @@ createTexture validation tests.
 TODO: review existing tests and merge with this plan:
 > All x= every texture format
 >
-> - {any dimension, mipLevelCount} = 0
```

Comment:
Not sure whether I misunderstood this. But I suppose we don't need to test different dimensions (texture sizes) * all formats. I think two tests to cover these two factors are enough.

Reply:
Got it. So I added a separate test named zero_size to cover this situation. PTAL.